modules/iasRtdProvider.js

```diff
@@ -51,94 +69,107 @@ function stringifyScreenSize() {
   return [(window.screen && window.screen.width) || -1, (window.screen && window.screen.height) || -1].join('.');
 }
 
-function getPageLevelKeywords(response) {
+function formatTargetingData(adUnit) {
   let result = {};
-  if (response.brandSafety) {
-    shallowMerge(result, response.brandSafety);
+  if (iasTargeting[BRAND_SAFETY_OBJECT_FIELD_NAME]) {
+    extend(result, iasTargeting[BRAND_SAFETY_OBJECT_FIELD_NAME]);
+  }
+  if (iasTargeting[FRAUD_FIELD_NAME]) {
+    result[FRAUD_FIELD_NAME] = iasTargeting[FRAUD_FIELD_NAME];
+  }
+  if (iasTargeting[CUSTOM_FIELD_NAME]) {
```

Comment:
Should this also be checking for the existence of `IAS_KW` in `iasTargeting[CUSTOM_FIELD_NAME]`?

Reply:
I don't see a code change to resolve this comment

Reply:
This comment still needs to be resolved with a code update

Reply:
@ncolletti I solved it.